openapi/p4pa-pu-bff.openapi.yaml

```
@@ -47,6 +47,22 @@ paths:
           description: Forbidden - Access denied
         '500':
           description: Internal Server Error
+  /health:
```

just health could be ambigous
```suggestion
  /core-health:
```